.travis.yml

```diff
@@ -6,14 +6,16 @@ node_js:
     - "4.1"
 
 before_install:
+    - rvm install 2.2.2
```

Comment:
Alternatively, it might be more travis-like to do something like:

```yaml
languages:
  - node_js
  - ruby

node_js:
  - 4.1

rvm:
  - 2.2.2
```